src/Checkpoints.jl

```diff
@@ -68,6 +68,22 @@ Returns a vector of all available (registered) checkpoints.
 """
 available() = collect(keys(CHECKPOINTS))
 
+"""
+    enabled() -> Vector{String}
+
+Returns a vector of all enabled ([`config`](@ref)ured) checkpoints.
+"""
+enabled() = filter(is_enabled, available())
+
+"""
+    is_enabled(name) -> Bool
```

Comment:
```suggestion
    _isenabled(name) -> Bool
```
i.e. no `_` to be consistent with julia convention, e.g. `isa isqrt isone isodd isnan isinf isdir iszero isreal`.

My preference is usually to keep the API as small as possible, because that reduces the probability of having to make a breaking change. So in this case I would just go with exporting `enabled()`. We can always export `_isenabled` later if there is a need.

p.s. we should `export enabled`

Reply:
> My preference is usually to keep the API as small as possible

That makes sense, I was thinking that we may as well make it "available" (not exported but not `_`'d either). But it's not needed yet.

I didn't `export enabled` by analogy to `available` not being exported. Does that change your view at all?

Reply:
Looks like `config` is not exported either (I think we should export it).

Let's export `enabled`, but not `available` (since there seems to be no use case for `available` outside of the package, but there is for `enabled`).

@oxinabox thoughts?

Reply:
A reason I see for not exporting these functions is readability in user code. It's not obvious that `config` is about checkpoints if the call is e.g. `config(["value1", "value2"], path)`. Even more so for `enabled()` on its own. If we exported it, I would prefer to call it `enabled_checkpoints`.

Reply:
@mzgubic let me know if it looks good now

Reply:
LGTM!